registration/registration-services/src/main/java/io/mosip/registration/dto/VersionMappings.java

```diff
@@ -0,0 +1,15 @@
+package io.mosip.registration.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class VersionMappings {

```

Comment:
Please add JsonIgnoreOnUnknown annotation to this DTO

Reply:
Done.